updating/index.adoc

```diff
@@ -0,0 +1,85 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
+== Understanding the OpenShift Update Service
+xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[About the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. 
+
+['updating-clusters-overview-install-and-configure-update-service']
+== Installing and configuring the OpenShift Update Service
```

Comment:
This part was moved into `Updating a restricted network cluster` in [v4.8](https://docs.openshift.com/container-platform/4.8/updating/updating-restricted-network-cluster.html).

Reply:
Addressed- Removed this individual section and added it under `Updating a restricted network cluster`

Reply:
ack